getting-started/installation/installation-installing-from-progress-control-panel.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Not related to the content of the article, but the images. I would recommend you to hide your email address. Anti-spam measure.

Reply:
Good point. Will update them :)